src/backend/Dockerfile.gisaid

```diff
@@ -57,7 +57,7 @@ RUN mkdir /ncov && \
     cd /ncov && \
     git init && \
     git remote add origin https://github.com/nextstrain/ncov.git && \
-    git fetch origin 6691370325e5980d4f6073f7672759711c305a05 && \
+    git fetch origin master && \
```

Comment:
Can we still pin to a specific github rev, or do we just want to accept any risks of being pinned to latest so we can be more responsive to omicron?

Reply:
Nevermind, I see Dan's earlier comments now. We can pin to master for awhile!

Reply:
I'd love to see what you guys think. Punting this to discuss in product meeting